db/migrate/20170109151629_fix_welsh_publisher_content_locales.rb

```diff
@@ -0,0 +1,59 @@
+class FixWelshPublisherContentLocales < ActiveRecord::Migration[5.0]
+  def up
+    welsh_slugs = [
+      "anfonwch-rhent-prydles-manylion",
+      "anghytuno-phenderfyniad-treth",
+      "archebu-prawf-gyrru-theori",
+      "archebu-prawf-gyrru-ymarferol",
+      "benthyciadau-trefnu",
+      "cerbydau-wediu-heithrio-o-dreth-cerbyd",
+      "chwilio-am-swydd",
+      "cofnodi-ple-ar-gyfer-trosedd-traffig",
+      "cofrestru-i-bleidleisio",
+      "costau-galwadau",
+      "credyd-pensiwn",
+      "cyfraddau-isafswm-cyflog-cenedlaethol",
+      "cyfriflen-pensiwn-y-wladwriaeth",
+      "cymraeg",
+      "cynghorydd-budd-daliadau",
+      "cysylltu-canolfan-byd-gwaith",
+      "cysylltwch-gwasanaeth-pensiwn",
+      "deall-eich-bil-treth-hunanasesiad",
+      "dod-o-hyd-i-fanylion-cyswllt-pensiwn",
+      "dod-o-hyd-i-utr-sydd-ar-goll",
+      "gwarchod-eich-tir-ach-eiddo-rhag-twyll",
+      "gwasanaethau-cthem-mewngofnodi-neu-gofrestru",
+      "gweld-neu-rannu-eich-gwybodaeth-trwydded-yrru",
+      "gwirio-statws-mot",
+      "gwirio-treth-cerbyd",
+      "gwneud-cais-am-lwfans-priodasol",
+      "gwneud-hos",
+      "gwybodaethamgofrestru",
+      "gwyliau-banc",
+      "hysbysebu-swydd",
+      "hysbysu-twyll-budd-daliadau",
+      "lawrlwytho-offer-twe-sylfaenol-cthem",
+      "lwfans-ceisio-gwaith",
+      "lwfans-gofalwr",
+      "lwfans-gofalwr-hysbysu-newidiadau",
+      "mynediad-at-waith",
+      "olrhain-cais-eich-trwydded-yrru",
+      "talu-dirwy-dvla",
+      "talu-dirwy-llys-ar-lein",
+      "talu-gorchymyn-cynhaliaeth-ar-lein",
+      "taluch-bil-treth-hunanasesiad",
+      "treth-car",
+      "treth-incwm",
+      "twyll-budd-dal",
+      "yswiriant-gwladol",
+    ]
+
+    welsh_slugs.each do |slug|
+      content_items = ContentItem.where(base_path: "/#{slug}")
```

Comment:
It'd be good to including the `publishing_app` in this where condition - just to ensure this migration doesn't accidentally affect content from any other publishing apps?